### ISSUE TITLE
moving bcup file and deleting archive

### ISSUE THREAD

**Issue by Li-An** [buttercup/buttercup-desktop#899]
Hello, Buttercup seems interesting but I struggle with simple things. Il created an archive and imported a keepass archive and it works. OK.
Then I  decided to move the bcup file to my Nextcloud directory. The archive in Buttercup desktop won’t open anymore. OK. I open a new achive and use the bcup moved file for it. But there is no entry showing.
And I cannot delete the old archive entry if the file is moved (it need to be here to delete the entry).

**Comment by sallar** [comment]
Hi @Li-An, after moving your archive file (.bcup) to Nextcloud, did you remove it from Buttercup desktop and add it again?

**Comment by Li-An** [comment]
Not sure I quite understand your question but:

* the bcup file was moved (not copied) and in this case there is no way to delete the archive entry (I have to place the bcup file at its original location to delete the archive entry)
* I made a new test. I deleted all archives entries. Close Buttercup. Launch Buttercup. It asks me to open an archive and I choose the bcup file in Nextcloud directory. I type the password. No entries are showing. It looks empty.
